PWGHF/Tasks/taskMcValidation.cxx

```diff
@@ -1167,7 +1168,7 @@ struct HfTaskMcValidationRec {
                           aod::TracksWMc const& mcTracks,
                           aod::McParticles const& mcParticles,
                           aod::McCollisions const& mcCollisions,
-                          aod::BCsWithTimestamps const& bcs,
+                          BCsInfo const& bcs,
```

Comment:
I'd call the `BCsInfo` data type if and only if UPC selections are enabled. This means that in the process functions that do not apply UPC selections I'd not use at all the data type `BCsInfo`, but I'd keep the data type `aod::BCsWithTimestamps`